spyvm/model.py

```diff
@@ -799,8 +871,11 @@ def str_content(self):
                 return "<omitted>"
         return "'%s'" % self.as_string().replace('\r', '\n')
 
-    @jit.look_inside_iff(lambda self: self._size < 256)
```

Comment:
I found this to appear in traces otherwise during lookup. If you think the other way is better, go ahead and try that.